### PR TITLE
updated dart pub get command

### PR DESCRIPTION
**Pull request by Abhishek01039**
`pub get` is deprecated.
https://dart.dev/tools/pub/cmd/pub-get

### CODE REVIEW

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -54,9 +54,9 @@ jobs:
       - run: dart --version
       - uses: actions/checkout@v2
       - id: pkg_web_css_pub_get
-        name: "pkg/web_css; pub get --no-precompile"
+        name: "pkg/web_css; dart pub get --no-precompile"
```
Comment (jonasfj) [inline .github/workflows/dart.yml:57]: This is a generated file, as it there is some support for old versions this haven't been migrated yet, see `mono_repo`

**Review by jonasfj — CHANGES_REQUESTED** [review]
If we avoid manual changes to `.github/workflows/dart.yml` then this might be  good change :D

File: .github/workflows/dart.yml (hunk 1 above)
Comment (Abhishek01039) [reply .github/workflows/dart.yml]: Ah Thank
@jonasfj

File: tool/ci.sh [hunk 2]
```diff
@@ -47,11 +47,11 @@ for PKG in ${PKGS}; do
     exit 64
   fi
 
-  pub get --no-precompile || EXIT_CODE=$?
+  dart pub get --no-precompile || EXIT_CODE=$?
```
Comment (jonasfj) [inline tool/ci.sh:50]: Top of this file says: `# Created with package:mono_repo v3.4.2`

I'm guessing there is little reason to modify it, maybe we should patch `mono_repo` to write more clearly not to modify such files, I assume `mono_repo` will overwrite it.. If not then changes in this file might make sense...

**Review by jonasfj — CHANGES_REQUESTED** [review]

File: tool/ci.sh (hunk 2 above)
Comment (Abhishek01039) [reply tool/ci.sh:50]: Should I discard this changes? @jonasfj

Comment (jonasfj) [reply tool/ci.sh:50]: yeah, let's not modify generated files.

Comment (Abhishek01039) [reply tool/ci.sh:50]: ok sure Thanks.

Comment (Abhishek01039) [reply tool/ci.sh]: Done @jonasfj

**Review by jonasfj — APPROVED** [review]

**Comment by Abhishek01039** [comment]
Thanks @jonasfj

**Pull request merged**